frontend/dockerfile/linter/linter.go

```diff
@@ -91,8 +91,12 @@ func (rule *LinterRule[F]) Run(warn LintWarnFunc, location []parser.Range, txt .
 	warn(rule.Name, rule.Description, rule.URL, short, location)
 }
 
-func LintFormatShort(rulename, msg string, startLine int) string {
-	return fmt.Sprintf("%s: %s (line %d)", rulename, msg, startLine)
+func LintFormatShort(rulename, msg string, line int) string {
+	msg = fmt.Sprintf("%s: %s", rulename, msg)
+	if line > 0 {
+		msg = fmt.Sprintf("%s (line %d)", msg, line)
```

Comment:
@tonistiigi Is it needed to have the line number in progress as we already have this information in?:

```
Usage of undefined variable '$PAHT' (did you mean $PATH?)
Dockerfile:3
--------------------
   1 |     FROM alpine
   2 | >>> ENV PATH=$PAHT:/foo
   3 |
--------------------
```

Mostly looking at https://github.com/docker/actions-toolkit/pull/365#issuecomment-2168855412

![image](https://github.com/moby/buildkit/assets/1951866/329a2e8a-1282-4ff0-9b20-f75716254a4d)

![image](https://github.com/moby/buildkit/assets/1951866/04f8432c-a6b4-4d7b-ba8f-7476686d5863)

Reply:
That is only with `--debug`. By default there is no source code view.

Reply:
I agree that this is not needed for example in your github actions excerpt. It should probably be added in buildx side for views that need it, not inside the string here.